docs/configuration.asciidoc

```diff
@@ -185,6 +185,41 @@ please https://github.com/elastic/apm-agent-nodejs/issues[open an issue].
 
 Note that not all core Node.js API's can be instrumented without the use of Async Hooks if running Node.js 8 or above.
 
+[[transaction-ignore-urls]]
+==== `transactionIgnoreUrls`
+
+* *Type:* Array
+* *Default:* `[]`
+* *Env:* `ELASTIC_TRANSACTION_IGNORE_URLS`
+
+Used to restrict requests for certain URLs from being instrumented.
+
+This property should be set to an array containing one or more string patterns.
+
+When an incoming HTTP request is detected, its URL will be tested against each element 
+in this list.  The `transactionIgnoreUrls` property supports exact string matches, and
+simple wildcard (`*`) matches.  You may turn on case sensitivity by using the 
+`(?+i)` prefix.
```

Comment:
It took me a LONG time to realize that this is case-*insensitive* by default.  

1. Eww, but that's the way it was specified.
2. I think we should have a clear "Matching is case-sensitive by default. Prefix your pattern with `(?-i)` to make the matching case-sensitive."
3. Do you really want to extend the spec by supporting `(?+i)`? At least the Python agent doesn't support that. I argue lower in this review that the code can be made simpler by only supporting `(?-i)` and not `(?+i)`.

Reply:
Hmm Re: `(?+i)` -- my memory is I saw that used in the Java Agent's wildcard implementation, and I included parsing it out in case someone was copying rules between agents. Going back now though I don't see it anywhere -- so good catch and a reminder for me meditate on the fallibility of memory  

i.e. I'll remove the code for handling the + case and be explicit in the language about what is and isn't supported. 

And yes -- I'm also looking forward to participating in spec discussions instead of implementing specs decided on before our time. :)